src/main/kotlin/CheckersGame.kt

```diff
@@ -0,0 +1,19 @@
+class CheckersGame (private val model : CheckersModel, playerWhite : BasePlayer, playerBlack : BasePlayer) {
+    private val players = Array<BasePlayer>(2) { i -> if (i == 0) playerWhite else playerBlack}
+
+    fun game() {
```

Comment:
Methods names should be verbs, `game` is a noun

Reply:
Fixed